src/ocm/components/featureSupportLevels/utils.tsx

```diff
@@ -53,3 +60,87 @@ export const getLimitedFeatureSupportLevels = (
     return undefined;
   }
 };
+
+const isArmSupported = (version: string, versions: OpenshiftVersionOptionType[]) => {
```

Comment:
What is the difference between `version` and `versions`?
Can we give `version` a name that aligns with its context?